src/modules/editor/selectors.ts

```diff
@@ -101,8 +112,26 @@ export const isFetching = createSelector<RootState, Project | null, boolean, Loa
   }
 )
 
-export const getVisibleItems = createSelector<RootState, string[], ItemState['data'], Item[]>(
+export const getVisibleItems = createSelector<RootState, string | null, boolean, string[], ItemState['data'], Item[]>(
```

Comment:
Would you mind adding tests for these new selectors?
You can always ask for the selector function when testing `createSelector`s to make the tests easier.

Reply:
done 👍